### DISCUSSION TITLE
Manual/HDRColorPicker

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#54] [Notes]
# Manual/HDRColorPicker

https://docs.unity3d.com/Manual/HDRColorPicker.html

**Comment by vertxxyz** [comment]
If you want to display this picker in a script, add a [`ColorUsage`](https://docs.unity3d.com/ScriptReference/ColorUsageAttribute.html) attribute to a [`Color`](https://docs.unity3d.com/ScriptReference/Color.html) member.
Similarly, for HDR colour pickers in shaders, you can add `[HDR]`, as specified [here](https://docs.unity3d.com/Manual/SL-Properties.html).
